genisys/main.py

```diff
@@ -13,15 +15,10 @@ def validate(modules):
 
 def install_config(file, root="/"):
     print(f"Installing config file: {file} with root at {root}")
-
-    raise NotImplementedError
-    # TODO: Implement the installation logic here
-
-def generate_config(file, root="/"):
-    print(f"Generating config file: {file} at directory {root}")
-
-    raise NotImplementedError
-    # TODO: Implement the generate logic here
+    ps.install(root)
```

Comment:
Looks like you're trying to run from the module directly here. You need to make an instance of each of these classes then call the `install` method on those instances.

Reply:
should I pass root into the constructor when I instantiate them?